pkg/distributor/writefailures/metrics.go

```diff
@@ -0,0 +1,29 @@
+package writefailures
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type metrics struct {
+	loggedCount    *prometheus.CounterVec
+	discardedCount *prometheus.CounterVec
+}
+
+func newMetrics(reg prometheus.Registerer, subsystem string) *metrics {
+	// prometheus.NewCounterVec(opts prometheus.CounterOpts, labelNames []string)
+	return &metrics{
+		loggedCount: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Namespace:   "loki",
+			Name:        "write_failures_logged_total",
+			Help:        "The total number log failures were logged for a tenant.",
```

Comment:
This help text does not make sense to me.

Reply:
I pushed a comment rewriting it a little bit, lmk if it looks better.